commit/plugin.go

```diff
@@ -105,18 +243,18 @@ func (p *Plugin) Close() error {
 	return nil
 }
 
-func (p *Plugin) decodeOutcome(outcome ocr3types.Outcome) (Outcome, State) {
+func (p *Plugin) decodeOutcome(outcome ocr3types.Outcome) Outcome {
```

Comment:
Might as well merge this function with `DecodeOutcome` now that the special part has been removed.

Reply:
It's called in multiple places so far. Nicer having it this way instead of copying the logic of empty outcome and the error checking 3 times IMO.